addons/web_editor/static/src/js/editor/odoo-editor/src/utils/sanitize.js

```diff
@@ -265,6 +260,14 @@ export function sanitize(nodeToSanitize, root = nodeToSanitize) {
             }
         }
 
+        // Sanitize the tree.
+        while (node?.isConnected && root.contains(node)) {
+            if (!isProtected(node)) {
+                node = sanitizeNode(node, root); // The node itself might be replaced during sanitization.
+            }
+            node = node.firstChild || node.nextSibling || ancestors(node, root).find(a => a.nextSibling)?.nextSibling;
+        }
```

Comment:
With the change you made for font awesome icons above, do you still need this change?

Reply:
We do not need it. I just removed that is pushed the code.